blade/procData/models.py

```diff
@@ -11,4 +11,17 @@ class Meta:
 		ordering = ['-id']
 
 	def __str__(self):
-		return self.email
\ No newline at end of file
+		return self.email
+
+
+#class results
+
+#class query
+class dbQueriedUsers(models.Model):
+	user = models.CharField(max_length=100,unique=True)
```

Comment:
is this 8 spaces? try to set your editor to use consistently 4 spaces

Reply:
actually when adding code with spaces python wouldn't like it so already replaced a couple of tabs in views (and admin): https://github.com/abaddonpuff/bladeART/pull/12/commits/79733ce9a94e19e093e1428e9431f77f2ae8d578